### PR TITLE
Tuple

### LINKED ISSUE #11: Tuple as a '*->*->*' kind
this might complicate things, but is worth bringing up as an idea:
maybe types like (a,b) should be Tuple a b, and then end up as '? ? ?' in the problem.

### PR DESCRIPTION
**Pull request by yehoshuapw**
closes #11 

this PR changes tuples to to `Pair` and `Triple`,
so for example: 
```
zip3      ❓❓❓❓ :: 🤷 🤷 -> 🤷 🤷 -> 🤷 🤷 -> 🤷 (🤷 🤷 🤷 🤷)
             List a -> List b -> List c -> List (Triple a b c) 
unzip3    ❓❓❓❓❓❓ :: 🤷 (🤷 🤷 🤷 🤷) -> 🤷 (🤷 🤷) (🤷 🤷) (🤷 🤷)
             List (Triple a b c) -> Triple (List a) (List b) (List c) 
quotRem   ❓❓❓❓❓❓❓ :: 🤷 🤷 => 🤷 -> 🤷 -> 🤷 🤷 🤷
             Integral a => a -> a -> Pair a a
```

The complexity I mentioned in the issue, is actually about the parentheses, but it actually works out.

### CODE REVIEW

**Comment by divarvel** [comment]
`Pair` is sometimes used for homogeneous pairs. What about `Tuple` and `Tuple3`, as is done in purescript?

**Comment by mossprescott** [comment]
At the risk of pointing out the obvious, the actual “names” for these types are `(,)` and `(,,)`.  If you use signatures like `fst :: (,) a b -> a` it would have the advantage of actually being valid code. 
But I can understand if people find that more confusing than fictional Pair/Tuple types.

**Comment by divarvel** [comment]
The parser doesn't handle symbolic names like `(,)`,  so `Tuple` has the benefit of not requiring further changes.

**Comment by yehoshuapw** [comment]
> `Pair` is sometimes used for homogeneous pairs. What about `Tuple` and `Tuple3`, as is done in purescript?

changed. If purescript does that, then it's closer then Pair.

File: public/index.html [hunk 1]
```diff
@@ -211,6 +211,7 @@
         </p>
         <ul>
             <li>Brackets (<code>[]</code>) are replaced with <code>List</code>;</li>
+            <li>Tuples (<code>(,)</code> and <code>(,,)</code>) are replaced with <code>Pair</code> and <code>Triple</code>;</li>
```
Comment (divarvel) [inline public/index.html]:
```suggestion
            <li>Tuples (<code>(,)</code> and <code>(,,)</code>) are replaced with <code>Tuple</code> and <code>Tuple3</code>;</li>
```

**Comment by divarvel** [comment]
Thanks! That's merged, i'll deploy shortly

**Pull request merged**